docs/source/Support/bskReleaseNotes.rst

```diff
@@ -84,6 +84,7 @@ Version  |release|
 - The BSK install instructions are updated to ask users to install by first ``pip`` installing build
   required packages through ``requirements_dev.txt``.
 - Update the build process to use ``conan`` version 2.x
+- Memory Leak for ``reactionWheelStateEffector`` fixed via destructor update, swig update, and removing ``.disown()`` in factory.
```

Comment:
You put this comment in the middle of a prior release notes comment that had a warning associated with it.  Please put it at the bottom of the list of the Current beta cycle.